README.md

```diff
@@ -3,6 +3,7 @@
 ![coverage][coverage_badge]
 [![style: very good analysis][very_good_analysis_badge]][very_good_analysis_link]
 [![License: MIT][license_badge]][license_link]
+[![Codemagic build status](https://api.codemagic.io/apps/667c744745b1b4753769d379/667c744745b1b4753769d378/status_badge.svg)](https://codemagic.io/app/667c744745b1b4753769d379/667c744745b1b4753769d378/latest_build)
```

Comment:
this is neat that you got this badge ... this might be good to note https://blog.codemagic.io/how-tuist-migrated-from-github-actions-to-codemagic-for-faster-and-more-reliable-ci/
and of course just the fact that we might not always use github and they have macs to build ipas.  devops is a whole thing, which i hope someone on our team will take an interest in.
Please make a jira ticket for beta app distribution ie see https://blog.codemagic.io/beta-testing-platforms-2019/

Reply:
Linked here: [WA-41](https://anirac-tech.atlassian.net/browse/WA-41?atlOrigin=eyJpIjoiZmZlMGVkZjc2ZDhiNGQ0M2E3NjE5YzQyN2NmMzI4MmEiLCJwIjoiaiJ9)